src/sign/utils.ts

```diff
@@ -14,13 +14,9 @@ export async function verify(address, sig, data, network = '1') {
   // console.log('Hash', hash);
   // console.log('Address', address);
 
-  try {
-    const recoverAddress = verifyTypedData(domain, types, message, sig);
-    // console.log('Recover address', recoverAddress);
-    if (address === recoverAddress) return true;
-  } catch (e) {
-    // console.log('Could not recoverAddress:' + e.message);
-  }
+  const recoverAddress = verifyTypedData(domain, types, message, sig);
```

Comment:
Wouldn't this function fail in the case where sig is `0x`?

Reply:
Opps correct reverted it